test/connectivity/modules/NetworkController/NetworkProfile.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    public class NetworkProfile
+    {
+        public string ProfileType { get; set; }
+
+        public ProfileSetting ProfileSettings { get; set; }
+
+        public class ProfileSetting
```

Comment:
Either rename this class as ProfileSettings or rename variable name from ProfileSettings to ProfileSetting.